README.md

```diff
@@ -37,6 +37,7 @@ Usage of _output/linux/amd64/kube-rbac-proxy:
       --config-file string                          Configuration file to configure kube-rbac-proxy.
       --insecure-listen-address string              The address the kube-rbac-proxy HTTP server should listen on.
       --kubeconfig string                           Path to a kubeconfig file, specifying how to connect to the API server. If unset, in-cluster configuration will be used
+      --stale-cache-interval string	                The interval to keep auth request review result for in case of unconsciousness of apiserver (default 0)
```

Comment:
Wouldn't it make sense, in order to tolerate some downtime, to have this cache default to let's say 10 minutes? Also the name doesn't reflect to me what this means. I would rather use the same naming as the code uses, something like `--token-cache-ttl`. What do you think?

Reply:
I described my thoughts about the purpose of this cache below. For now, I have no idea which name to give to the option.